# Patch to set the new end dependant detachment to 1
# Contribution from the Crick HPC team
# uploaded by J. Sassmannshausen
diff -Nru cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b-orig/src/sim/hand_prop.h cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b/src/sim/hand_prop.h
--- cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b-orig/src/sim/hand_prop.h	2019-01-17 09:14:25.000000000 +0000
+++ cytosim-3ecab2584958081f0bbb42642e849c84f7d34c0b/src/sim/hand_prop.h	2019-03-07 09:03:53.027352000 +0000
@@ -21,7 +21,7 @@
  of dynamic microtubules and mitotic motors" published in 2018
  By J. Roostalu, J. Rickman, C. Thomas, F. Nedelec and T. Surrey
  */
-#define NEW_END_DEPENDENT_DETACHMENT 0
+#define NEW_END_DEPENDENT_DETACHMENT 1
 
 
 /// Property for Hand
